### ISSUE TITLE
error in running compiled Zgoubi, related to pariz.nml

### ISSUE THREAD

**Issue by bnash** [radiasoft/zgoubi#57]
I am trying to add additional tests to the ctest infrastructure.  I am working on the branch add-zgoubi-tests.  I got an updated pull from the repository and built using cmake, and then make.
Now I go to run zgoubi in one of the KEYWORDS directories with a zgoubi.res file.
I get the error:

```
ERROR STOP  Assertion 'initialize_input_parameters: pariz.nml opened.' failed on image            1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        

Error termination. Backtrace:
#0  0x7fe7772fff9a in ???
#1  0x7fe777300b85 in ???
#2  0x7fe777301c36 in ???
#3  0x54e3c5 in ???
#4  0x40294a in __pariz_namelist_interface_MOD_initialize_input_parameters
	at /home/vagrant/src/radiasoft/zgoubi-rs/zgoubi/pariz_namelist_implementation.f90:24
#5  0x545e2f in zgoubi_main
	at /home/vagrant/src/radiasoft/zgoubi-rs/zgoubi/zgoubi_main.f:82
#6  0x401cec in main
	at /home/vagrant/src/radiasoft/zgoubi-rs/zgoubi/zgoubi_main.f:26
```

**Comment by bnash** [comment]
Now I see that I need to have the file pariz.nml in the directory I am running zgoubi from.

**Comment by rouson** [comment]
@bnash Is it working now?  If so, feel free to close this issue.  While you're working on tests, it would be great to have a test that exercises zbougi's BLAS/LAPACK dependency. There's a reasonable chance that such a test will fail when building with a compiler other than the one that was used to build the system BLAS/LAPACK libraries.

**Comment by dtabell** [comment]
@Boaz: I think you should not see _anything_ about pariz.nml. The add-zgoubi-tests branch, the one I think you’re working in, should not include any of the DMA work that Damian has done.

**Comment by bnash** [comment]
Hi @rouson and @dtabell,
Yes, as long as I include pariz.nml in each run directory, zgoubi works correctly.
But, as @dtabell says, I'm also confused on why this DMA development is in the  add-zgoubi-tests branch.  Perhaps I have an error in my git configuration or used the wrong command.
I'm still trying to learn a few things about working with branches.

**Comment by bnash** [comment]
@rouson, I'll look into the BLAS/LAPACK dependency and try to find tests that use this.

**Comment by dtabell** [comment]
@Boaz, Would you please verify that you are working in the branch add-zgoubi-tests?

**Comment by bnash** [comment]
@dtabell, yes, I believe so.
Typing 
`> git branch`
returns
```
* add-zgoubi-tests
  master
```

**Comment by dtabell** [comment]
@bnash: `git status` should tell you the same information.

**Comment by bnash** [comment]
Yes, `git status` returns
```
On branch add-zgoubi-tests
Your branch is up-to-date with 'origin/add-zgoubi-tests'.

nothing to commit, working tree clean
```

**Comment by dtabell** [comment]
So, yes, @rouson needs to have a look at this.
FYI ... I see the following in the network graph:
<img width="747" alt="image" src="https://user-images.githubusercontent.com/11483676/51493319-f4b57f00-1d82-11e9-846c-d635a8eb110f.png">

The green branch coming off add-zgoubi-tests is my fix-common-blocks branch. Since I don't see any extension of add-zgoubi-tests beyond that, I'm very puzzled! (The dates running along the top are in December 2018.)

**Comment by zbeekman** [comment]
FYI: Merging PR #52 on the 12th (December, I think) is responsible for incorporating newer developments into the `add-zgoubi-tests` branch. This can probably be undone, and a more-careful incorporation of the intended updates ported onto this branch.

**Comment by dtabell** [comment]
The last part of this work will be addressed by #84.
